script.js

```diff
@@ -1,4 +1,165 @@
+// game mode
+var gameMode = "p1Input";
+var player1Number = null;
+var player2Number = null;
+var player1Rolls = [];
+var player2Rolls = [];
+var player1Numbers = [];
+var player2Numbers = [];
+var roundNumber = 1;
+var player1Sum = 0;
+var player2Sum = 0;
+
+
+var rollDice = function () {
+  // produces a float between 0 and 7
+  var randomFloat = Math.random() * 6;
+  // take off the decimal
+  var resultInteger = Math.floor(randomFloat) + 1;
+  lastRoll = resultInteger;
+  console.log("dice roll = " + resultInteger)
+  return resultInteger;
+  
+};
+
+var playerRoll = function() {
+  var roll1 = rollDice();
+  var roll2 = rollDice(); 
```

Comment:
You can consider using a loop here. As mentioned yesterday, since the More Comfortable part allows players to roll more than 2 die, you want your code to be as flexible and future-proof as possible.